core/core-dotspacemacs.el

```diff
@@ -222,9 +222,10 @@ can be toggled through `toggle-transparency'.")
   "If non nil unicode symbols are displayed in the mode-line (eg. for lighters)")
 
 (defvar dotspacemacs-smooth-scrolling t
-  "If non nil smooth scrolling (native-scrolling) is enabled. Smooth scrolling
-overrides the default behavior of Emacs which recenters the point when
-it reaches the top or bottom of the screen.")
+  "If non nil smooth scrolling (native-scrolling) is enabled.
+Smooth scrolling overrides the default behavior of Emacs which
+recenters point when it reaches the top or bottom of the
```

Comment:
You probably need to synchronize `core/templates/.spacemacs.template` to keep both synchronized :-)